File: Project1/starter/Game.java
import javax.swing.JFrame;
import java.awt.Toolkit;

public class Game extends JFrame {
	Model model;
	Controller controller;
	View view;

	public Game() {
		model = new Model();
		controller = new Controller(model);
		view = new View(controller, model);
		this.setTitle("Turtle attack!");
		this.setSize(500, 500);
		this.setFocusable(true);
		this.getContentPane().add(view);
		this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
		this.setVisible(true);
		this.addMouseListener(controller);
		this.addKeyListener(controller);
	}

	public static void main(String[] args) {
		Game g = new Game();
		g.run();
	}

	public void run() {
		while (true) {
			controller.update();
			model.update();
			view.repaint();
			Toolkit.getDefaultToolkit().sync();

			try {
				Thread.sleep(40);
			} catch (Exception e) {
				e.printStackTrace();
				System.exit(1);
			}
		}
	}
}

File: game.py
import pygame
import time
from model import Model
from controller import Controller
from view import View
from pygame.locals import *
from time import sleep


class Game:
  def __init__(self):
    pygame.init()
    self.model = Model()
    self.view = View(self.model)
    self.controller = Controller(self.model)

  def run(self):
    while self.controller.keep_going:
      self.controller.update()
      self.model.update()
      self.view.update()
      sleep(0.01)


g = Game()
g.run()